src/extensions/default/PrefsCodeHints/main.js

```diff
@@ -38,11 +38,24 @@ define(function (require, exports, module) {
         JSONUtils           = brackets.getModule("language/JSONUtils"),
         Strings             = brackets.getModule("strings"),
         ThemeManager        = brackets.getModule("view/ThemeManager"),
+        CodeInspection      = brackets.getModule("language/CodeInspection"),
         _                   = brackets.getModule("thirdparty/lodash"),
-        data                = JSON.parse(require("text!data.json")),
+        languages           = LanguageManager.getLanguages(),
```

Comment:
`statusChange` get fired every time an extension get loaded. We would need an event that is fired only when an extension gets installed/uninstalled, so we can update our preferences data.